yang/frr-zebra.yang

```diff
@@ -2920,6 +2920,15 @@ module frr-zebra {
       description
         "Enable PTM globally.";
     }
+    leaf route-map-delay {
+      type uint32 {
+        range "0..600";
```

Comment:
I don't like ranges, especially ones that seem arbitrary. Can this be removed?

Reply:
By removing ranges we'll allow values that were not allowed before to be set using the mgmtd API, even if we leave the CLI intact. So it's not safe to do this without checking that the code actually support bigger values, which I do not intend to do in this PR. I think relaxing the restrictions later if necessary won't be a breaking change, but it will definitely be a breaking change the other way around.